lib/readline.js

```diff
@@ -429,7 +428,9 @@ Interface.prototype._tabComplete = function() {
           }
         }
         handleGroup(self, group, width, maxColumns);
+      }
 
+      {
```

Comment:
Do we need `{` block `}` [here](https://github.com/addaleax/node/blob/37229c6bf7d911cadb1198a6c482ad489979a679/lib/readline.js#L433-L442) ? Seems unnecessary